### PR TITLE
task-6 add redux with CRUD, add filtering and sorting

### PR DESCRIPTION
**Pull request by AlexUCM**

### CODE REVIEW

File: src/components/Button.jsx [hunk 1]
```diff
@@ -52,10 +52,14 @@ const StyledButton = styled.button`
 `;
 
 export const Button = (props) => (
```
Comment (omelenetskiy) [inline src/components/Button.jsx:54]: It will be cleaner if you will use destructing here

File: src/components/MoviesList.jsx [hunk 2]
```diff
@@ -34,47 +37,87 @@ const ResultNumber = styled.span`
   margin: 0 5px 0 0;
 `;
 
+//
```
Comment (omelenetskiy) [inline src/components/MoviesList.jsx:40]: Useless comment

**Review by omelenetskiy — APPROVED** [review]

File: src/components/MoviesList.jsx (hunk 2 above)
Comment (AlexUCM) [reply src/components/MoviesList.jsx:40]: fixed in the next task

File: src/components/Button.jsx (hunk 1 above)
Comment (AlexUCM) [reply src/components/Button.jsx:54]: fixed in the next task

**Pull request merged**